doc/Programming-guide/Key-platform-features/Common-Type-Library/index.md

```diff
@@ -33,38 +67,105 @@ Matching FQN in a different Tenant | N/A | OK | OK
 Matching FQN at Application level | error | error | error unless the version is unique
```

Comment:
"Each CT is identified by its unique Fully Qualified Name (**FQN**) and is subject to versioning."
- redundant